src/TypeScript.UnitTests/Analyzer/TypeScriptAnalyzerTests.cs

```diff
@@ -242,7 +242,26 @@ public async Task ExecuteAnalysis_AnalysisFinished_NotifiesThatAnalysisFinished(
             statusNotifier.VerifyNoOtherCalls();
         }
 
-        private Mock<IEslintBridgeAnalyzer> SetupEslintBridgeAnalyzer(IReadOnlyCollection<IAnalysisIssue> issues = null, Exception exceptionToThrow = null)
+        [TestMethod]
```

Comment:
This test is the same as in the PR for Remove TaskScheduler.Default from JavaScriptAnalyzer